octoprint_mrbeam/camera/corners.py

```diff
@@ -20,9 +20,9 @@
 from numpy.linalg import norm
 import cv2
 from octoprint_mrbeam.camera import lens
+from octoprint_mrbeam.mrb_logger import mrb_logger
 
-# Set this after merging the logging overhaul.
-_logger = logging.getLogger(__name__)
+_logger = mrb_logger("octoprint.plugins.mrbeam.camera.corners")
```

Comment:
Please keep the consistent `__name__` scheme

Reply:
this leads to inconsistent log output. 
`__name__` results in `octoprint_mrbeam.camera.corners` while all the other mr beam output is this:
`octoprint.plugins.mrbeam.iobeam.lidhandler`

Reply:
I agree that `__name__` is nicer and that even it's output is shorter (= nicer). @amrsoll would you like to go though the code and change all logger names at once? (after the next stable?)

Reply:
> Also the deprecated values that you do find in the yaml file is just carried over from the previous version, they are not actively saved.

Yes, that's why we need some basic documentation.

Reply:
Yeah sure I can change the naming later :+1:

Reply:
ok - we don't have documentation on those yet.